drivers/usb/serial/xr_usb_serial_common.c

```diff
@@ -64,12 +64,25 @@
 #include <linux/compat.h>
 #endif
 
+#undef CONFIG_GPIOLIB
```

Comment:
why do you need this undef?

Reply:
@rene ah, I had a comment in commit message but removed it by mistake. I'll update it. Basically GPIO functionality interferes with RS485 mode because we use GPIO 5 to  control RX/TX. and I'd like to avoid it at any cost. Besides this part is not written 100% correctly and produces confusing error messages in demsg